src/Core/Domain/Product/Combination/QueryResult/EditableCombinationForListing.php

```diff
@@ -94,7 +100,8 @@ public function __construct(
         bool $default,
         DecimalNumber $impactOnPrice,
         int $quantity,
-        string $imageUrl
+        string $imageUrl,
+        float $ecoTax
```

Comment:
no floats.
Use DecimalNumber

Reply:
Agreed, float lack precision natively so we try and rely on DecimalNumber every time we can Especially on new code, or else we create additional technical debts because the purpose is to rely on DecimalNumber everywhere we have float values